nwbinspector/dandi.inspector_config.yaml

```
@@ -0,0 +1,7 @@
+CRITICAL:
+  - check_subject_exists
+  - check_subject_id_exists
+BEST_PRACTICE_VIOLATIONS:
```

```suggestion
BEST_PRACTICE_VIOLATION:
```

Same thing from before

I also assume SKIP is valid for the YAML?

Also, JSON configs can be validated against the schema; is there an analog of that for YAML?